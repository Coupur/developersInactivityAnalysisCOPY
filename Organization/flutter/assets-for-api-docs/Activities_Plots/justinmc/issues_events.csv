id,date,event,creator_login
2028198583,2018-12-15 00:29:20,assigned,justinmc
2031371640,2018-12-17 21:02:15,head_ref_force_pushed,justinmc
2031431699,2018-12-17 21:30:19,head_ref_force_pushed,justinmc
2033886966,2018-12-18 19:26:25,head_ref_force_pushed,justinmc
