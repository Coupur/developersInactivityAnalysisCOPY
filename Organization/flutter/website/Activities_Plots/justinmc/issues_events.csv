id,date,event,creator_login
2339953331,2019-05-14 12:34:38,mentioned,justinmc
2339953333,2019-05-14 12:34:38,subscribed,justinmc
