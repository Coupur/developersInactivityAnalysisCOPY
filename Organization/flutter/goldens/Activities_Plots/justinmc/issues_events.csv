id,date,event,creator_login
2359842338,2019-05-22 15:59:37,review_requested,justinmc
2359842339,2019-05-22 15:59:37,review_requested,justinmc
2359842350,2019-05-22 15:59:37,assigned,justinmc
2359865264,2019-05-22 16:07:22,referenced,justinmc
2359865271,2019-05-22 16:07:22,merged,justinmc
2359865293,2019-05-22 16:07:22,closed,justinmc
2359865510,2019-05-22 16:07:26,head_ref_deleted,justinmc
2357003968,2019-05-21 18:03:23,review_requested,justinmc
2357003977,2019-05-21 18:03:24,assigned,justinmc
2357283654,2019-05-21 19:55:37,review_requested,justinmc
2357313086,2019-05-21 20:07:37,head_ref_force_pushed,justinmc
2357406651,2019-05-21 20:44:19,head_ref_force_pushed,justinmc
2357409623,2019-05-21 20:45:23,head_ref_force_pushed,justinmc
2357410940,2019-05-21 20:45:51,head_ref_force_pushed,justinmc
2357708736,2019-05-21 23:23:18,referenced,justinmc
2357708738,2019-05-21 23:23:18,merged,justinmc
2357708743,2019-05-21 23:23:18,closed,justinmc
2357708812,2019-05-21 23:23:21,head_ref_deleted,justinmc
